content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "New design and UX digest"
+tags:
+- core
+- developer
+author: scherler
+---
+
+== Overview
+
+The recent changes to some core pages provide new design and UX and further dropping all external dependencies to prevent
+any possible malicious javascript introduced by third party libraries. https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5[Why?]
+
+This meant to drop the usage of the jelly layout lib (aka `xmlns:l="/lib/layout"`) and as well the page decorators it
+supported. However there is a new SimplePageDecorator which can be used to modify the look and feel for the login and sign up page.
+
+The following pages have given a new design:
+
+- HudsonIsLoading / HudsonIsRestarting
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_a.png[role="center"]
```

Comment:
Would it be possible to redo screenshots with browser pages with smaller height? Half of the blog post is empty space from screenshots

Reply:
Yeah will do so.

Reply:
@oleg-nenashev I have done that now